Examples/Python/src/Geometry.cpp

```diff
@@ -64,6 +77,18 @@ void addGeometry(Context& ctx) {
         .value("Other", Acts::Surface::SurfaceType::Other);
   }
 
+  {
+    py::enum_<Acts::VolumeBounds::BoundsType>(m, "VolumeType")
+        .value("Cone", Acts::VolumeBounds::BoundsType::eCone)
+        .value("Cuboid", Acts::VolumeBounds::BoundsType::eCuboid)
+        .value("CutoutCylinder",
+               Acts::VolumeBounds::BoundsType::eCutoutCylinder)
+        .value("Cylinder", Acts::VolumeBounds::BoundsType::eCylinder)
+        .value("GenericCuboid", Acts::VolumeBounds::BoundsType::eGenericCuboid)
+        .value("Trapezoid", Acts::VolumeBounds::BoundsType::eTrapezoid)
+        .value("Other", Acts::VolumeBounds::BoundsType::eOther);
+  }
```

Comment:
```suggestion
  py::enum_<Acts::VolumeBounds::BoundsType>(m, "VolumeType")
      .value("Cone", Acts::VolumeBounds::BoundsType::eCone)
      .value("Cuboid", Acts::VolumeBounds::BoundsType::eCuboid)
      .value("CutoutCylinder",
             Acts::VolumeBounds::BoundsType::eCutoutCylinder)
      .value("Cylinder", Acts::VolumeBounds::BoundsType::eCylinder)
      .value("GenericCuboid", Acts::VolumeBounds::BoundsType::eGenericCuboid)
      .value("Trapezoid", Acts::VolumeBounds::BoundsType::eTrapezoid)
      .value("Other", Acts::VolumeBounds::BoundsType::eOther);
```

Reply:
This I leave for consistency with the Surface enums.